bitnami/etcd/templates/statefulset.yaml

```diff
@@ -217,6 +217,10 @@ spec:
             - name: ETCD_INIT_SNAPSHOT_FILENAME
               value: {{ .Values.startFromSnapshot.snapshotFilename | quote }}
             {{- end }}
+            {{- if and .Values.disasterRecovery.enabled .Values.startFromSnapshot.enabled (not .Values.disasterRecovery.pvc.existingClaim) }}
+            - name: ETCD_INIT_SNAPSHOTS_DIR
+              value: "/snapshots"
```

Comment:
```suggestion
            - name: ETCD_INIT_SNAPSHOTS_DIR
              value: {{ ternary "/snapshots" "/init-snapshot" (and .Values.disasterRecovery.enabled (not .Values.disasterRecovery.pvc.existingClaim)) | quote }}
```

Reply:
What do you think about this proposal? This way we always align `ETCD_INIT_SNAPSHOTS_DIR` with the "volumeMounts" used in the chart (regardless the default value used in the image).

Reply:
Sure, it's better solution